docs/Config.md

```diff
@@ -230,7 +231,7 @@ Lazygit will edit a file with the first set editor in the following:
 
 ```yaml
 os:
-  editCommand: 'vim' # as an example
+  editor: 'vim' # as an example
```

Comment:
am I right in thinking this is a non-backwards compatible change? Given that people would currently have `editCommand: 'vim'` meaning it won't actually open the file? We'll need to ensure that this is backwards compatible somehow

Reply:
Added editCommandTemplate for backward compatibility, without changing the name of editCommand.